include/seqan/arg_parse/argument_parser.h

```diff
@@ -221,6 +227,21 @@ class ArgumentParser
                                         "FORMAT"));
         hideOption(*this, "export-help", true);
         setValidValues(*this, "export-help", "html man txt");
+
+        addOption(*this, ArgParseOption("",
+                                        "version-check",
+                                        "Choose OFF if you don't want any notifications. "
+                                        "Choose APP_ONLY if you don't want any notification about a new SeqAn version. "
+                                        "DEV is the default value and will check for both versions.",
```

Comment:
Maybe we rewrite this. I think that people might not understand the difference between app_only and dev.
Maybe something like:
```
Choose OFF to disable any update notifications. 
With  APP_ONLY you'll receive update notifications about new versions of your app. 
In DEV mode you'll receive update notifications regarding new versions of your app 
and new versions of the SeqAn library.
```